Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM rust:latest as builder
+
+WORKDIR /build
+
+COPY Cargo.toml Cargo.lock ./
+COPY src ./src
+
+RUN cargo fetch
+RUN cargo build --release
+
+FROM debian:bookworm-slim as dependencies
```

Comment:
How come this is `as dependencies` then later `as runtime` wouldn't it be the same to just have it `as runtime` here?

Reply:
You right, it's basically the same. I've removed the dependencies stage